drmemory/suppress-default.win.txt

```diff
@@ -832,6 +832,14 @@ KERNEL32.dll!...
 pdh.dll!*
 
 ##################################################
+# i#1301: Tcpip6_WSHGetSockaddrType final 9 EA bytes are left uninit
```

Comment:
Please update https://github.com/DynamoRIO/drmemory/issues/1302 with a callstack for the Win8.1 error you're seeing